src/Encamina.Enmarcha.SemanticKernel.Abstractions/IMemoryStoreExtender.cs

```diff
@@ -10,19 +10,25 @@ namespace Encamina.Enmarcha.SemanticKernel.Abstractions;
 /// <summary>
 /// Represents a memory manager (with some CRUD operations) over memories with multiple chunks from an <see cref="IMemoryStore"/>.
 /// </summary>
-public interface IMemoryManager
+public interface IMemoryStoreExtender
 {
     /// <summary>
-    /// This event is fired when <see cref="IMemoryManager"/> executes an action.
-    /// The event information is defined in the object <see cref="MemoryManagerEventArgs"/>, and the action types in the object <see cref="MemoryManagerEventTypes"/>.
+    /// This event is fired when <see cref="IMemoryStore"/> executes an action.
+    /// The event information is defined in the object <see cref="MemoryStoreEventArgs"/>, and the action types in the object <see cref="MemoryStoreEventTypes"/>.
     /// </summary>
-    event EventHandler<MemoryManagerEventArgs> MemoryManagerEvent;
+    event EventHandler<MemoryStoreEventArgs> MemoryStoreEvent;
 
     /// <summary>
-    /// Gets the instance of the memory store manage by this manager.
+    /// Gets the instance of the <see cref="IMemoryStore"/> object for this extender class.
     /// </summary>
     IMemoryStore MemoryStore { get; }
 
+    /// <summary>
+    /// This action fired a <see cref="MemoryStoreEvent"/> event when is invoked.
+    /// </summary>
+    /// <param name="e">The object <see cref="MemoryStoreEventArgs"/> contains the event arguments for this event.</param>
+    void OnMemoryStore(MemoryStoreEventArgs e);
```

Comment:
The name does not follows C# trigger events naming conventions. It should be `RaiseMemoryStoreEvent` 

Reference: https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/event

Reply:
Thank you for your contribution. You are right. I will update it.